css/style.css

```diff
@@ -1,19 +1,19 @@
-/* 
 
-Style Hints: 
+Body{
+	Font Size = 14px
```

Comment:
CSS properties should always follow this syntax: `font-size: 14px;`. You used `=` and no semi-colons at points, which will cause you problems.

Also make sure to name your properties properly (i.e., `font-family` instead of `Font Family`).